.eslintrc.json

```diff
@@ -0,0 +1,9 @@
+{
+    "extends": "standard",
```

Comment:
We should use 2 spaces in this file since that's what we use elsewhere

Reply:
Oh, yeah, my Atom tabs are a little messed up. I'll fix that haha.